### ISSUE TITLE
Feature Request: Add engine mode for global database to aws_rds_cluster 

### ISSUE THREAD

**Issue by shawnfeldman** [terraform-providers/terraform-provider-aws#6846]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

Global databases were added to aurora to make cross region replication easier to implement and manage. https://aws.amazon.com/rds/aurora/global-database/. 

### New or Affected Resource(s)

 
* aws_rds_cluster

### Potential Terraform Configuration

 

```hcl
resource "aws_rds_cluster" "default" {
  cluster_identifier      = "aurora-cluster-demo"
  availability_zones      = ["us-west-2a", "us-west-2b", "us-west-2c"]
  regions = ["us-east-1","us-west-2"]
  database_name           = "mydb"
  master_username         = "foo"
  master_password         = "bar"
  backup_retention_period = 5
  preferred_backup_window = "07:00-09:00"
  engine                  = "aurora"
  engine_mode             = "global"
}

```

### References

* https://www.terraform.io/docs/providers/aws/r/rds_cluster.html#engine_mode
* https://aws.amazon.com/rds/aurora/global-database/
* #0000

**Comment by bflad** [comment]
Support for `engine_mode` set to `global` within the `aws_rds_cluster` resource, new `global_cluster_identifier` argument within the `aws_rds_cluster` resource, and a new `aws_rds_global_cluster` resource submitted: #6861

**Comment by bflad** [comment]
Initial support for Aurora global databases has been merged and will release with version 1.53.0 of the AWS provider, likely tomorrow. 👍

**Comment by bflad** [comment]
This has been released in [version 1.53.0 of the AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#1530-december-20-2018). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.
